framework/src/main/java/org/apache/drill/test/framework/DrillTestOdbc.java

```diff
@@ -71,20 +85,34 @@ public void run() {
     		+ modeler.script + " "
     		+ modeler.queryFilename + " "
     		+ outputFilename;
-	LOG.info("Running test " + command);
+    LOG.info("Running test " + command);
     try {
 
   	  cmdConsOut = Utils.execCmd(command);
-  	  LOG.info(cmdConsOut.consoleOut);
+          LOG.info(cmdConsOut.consoleOut);
 
+      queries = Utils.getSqlStatements(modeler.queryFilename);
+      mainQueryIndex = queries.length / 2; // Currently, the main query must be in the middle of the list of queries
```

Comment:
Can we not have this restriction?

Reply:
@cchang738 , you mean we could use the @test notation instead right?